.circleci/config.yml

```
@@ -341,18 +341,128 @@ version: 2.1
 workflows:
     ademe:
         jobs:
-            - no-change:
+            - check-changelog:
+                filters:
+                    branches:
+                        ignore: main
+                name: check-changelog-ademe
+                site: ademe
+            - lint-changelog:
                 filters:
+                    branches:
+                        ignore: main
                     tags:
-                        only: /.*/
-                name: no-change-ademe
+                        only: /ademe-.*/
+                name: lint-changelog--ademe
+                site: ademe
+            - build-front-production:
+                filters:
+                    tags:
+                        only: /ademe-.*/
+                name: build-front-production-ademe
+                site: ademe
+            - lint-front:
+                filters:
+                    tags:
+                        only: /ademe-.*/
+                name: lint-front-ademe
+                requires:
+                    - build-front-production-ademe
+                site: ademe
+            - build-back:
+                filters:
+                    tags:
+                        only: /ademe-.*/
+                name: build-back-ademe
+                site: ademe
+            - lint-back:
+                filters:
+                    tags:
+                        only: /ademe-.*/
+                name: lint-back-ademe
+                requires:
+                    - build-back-ademe
+                site: ademe
+            - test-back:
+                filters:
+                    tags:
+                        only: /ademe-.*/
+                name: test-back-ademe
+                requires:
+                    - build-back-ademe
+                site: ademe
+            - hub:
+                filters:
+                    tags:
+                        only: /^ademe-.*/
```

```suggestion
                        only: /^ademe-.*/
```
why this filter compare to other has a ^ before?

---

This is regex we want only trigger this job for tag which starts with `ademe-`

---

I know that, but it's not written the same way for other jobs, it's written without ^, i think it's better to keep the same way of writing the rule